src/ui/documents/components/documentCard.tsx

```diff
@@ -1,26 +1,42 @@
 import React from 'react'
-import { View, StyleSheet, Text, Image, ImageBackground } from 'react-native'
+import {
+  View,
+  StyleSheet,
+  Text,
+  Image,
+  ImageBackground,
+  Dimensions,
+} from 'react-native'
 import { DocumentValiditySummary } from './documentValidity'
-import { DecoratedClaims } from 'src/reducers/account'
 import { ClaimInterface } from 'cred-types-jolocom-core'
-import { Colors, Typography, Spacing } from 'src/styles'
+import { Colors, Typography, Spacing, Typefaces } from 'src/styles'
+import { CredentialOfferRenderInfo } from 'jolocom-lib/js/interactionTokens/interactionTokens.types'
+import { fontMedium } from '../../../styles/typography'
+import { InvalidDocumentBackground } from '../../../resources'
 
-export const DOCUMENT_CARD_HEIGHT = 176
-export const DOCUMENT_CARD_WIDTH = 276
+const { width: WIDTH } = Dimensions.get('window')
+// TODO @clauxx rename
+const DOCUMENT_MARGIN = 20
```

Comment:
Why not just make this 40? :D

Reply:
So it's clear that the card dimensions are determined by the margin from the edge. I put it there just to be explicit.